django_esidoc/utils.py

```diff
@@ -10,6 +10,13 @@
 ENT_CORRELYCE_BASE_URL = getattr(settings, "ENT_CORRELYCE_BASE_URL", "")
 ENT_QUERY_STRING_TRIGGER = getattr(settings, "ENT_QUERY_STRING_TRIGGER", "sso_id")
 
+BASE_URLS = {
+    "ESIDOC": ENT_ESIDOC_BASE_URL,
+    "OCCITANIE": ENT_OCCITANIE_BASE_URL,
+    "OCCITANIEAGR": ENT_OCCITANIEAGR_BASE_URL,
+    "CORRELYCE": ENT_CORRELYCE_BASE_URL,
+}
```

Comment:
I think you are right. The dictionnary is better.

Regarding your comment on the fallback I agree with you.
We'll need to add `"HDF": ENT_HDF_BASE_URL,` to the `BASE_URLS` constant.